src/bindings/js.ts

```diff
@@ -976,8 +976,9 @@ export class JSBuilder extends ExportsWalker {
       }
       sb.push(`} = await (async url => instantiate(
   await (async () => {
-    try { return await globalThis.WebAssembly.compileStreaming(globalThis.fetch(url)); }
-    catch { return globalThis.WebAssembly.compile(await (await import("node:fs/promises")).readFile(url)); }
+    const isNodeOrBun = typeof process != "undefined" && process.versions != null && (process.versions.node != null || process.versions.bun != null);
```

Comment:
```suggestion
    const isNodeOrBun = typeof process != "undefined" && process.versions && (process.versions.node || process.versions.bun);
```

At some point in the future, I think this check can be removed, because Node.js in recent versions has `fetch` built-in.

Reply:
agree, let us wait for the new features in nodejs.